### ISSUE TITLE
Knife changer

### ISSUE THREAD

**Issue by Keko94** [iisp33dii/si0n-Source#3]
Knife changer doesn't apply skin, only model is modified

**Comment by iWonAMathsDebate** [comment]
Is there a way to fix it?

**Comment by iisp33dii** [comment]
Working fine for me: http://prntscr.com/dygaw3

**Comment by iWonAMathsDebate** [comment]
Well, for me its kinda halfway bugged. It mostly gives me the changed model without skin if i use scrollwheel and not the number 3-key, and sometimes it gives me the skin if im lucky and dont crash, and sometimes it just gives me default knife. Kinda weird that it works for you tho. Maybe I fucked up my game idk
take a look: https://gyazo.com/09e78041772282d404fa6829b1364a3b I got the skin once there, but later on it crashed
